templates/azure/ebook.html

```diff
@@ -0,0 +1,96 @@
+{% extends "azure/base_azure.html" %}
+
+{% block title %}Why do Cloud innovators migrate to Pro?{% endblock %}
+{% block meta_description %}{% endblock %}
+{% block meta_copydoc %}{% endblock meta_copydoc %}
+
+{% block head_extra %}
+  <style>
+    h2,
+    .p-heading--2 {
+      font-weight: 300;
+    }
+  </style>
+{% endblock %}
+
+{% block content %}
+
+<section class="p-strip--suru-bottomed">
+  <div class="row">
+    <div class="col-7 u-vertically-center">
+      <div>
+        <h1>Why do Cloud innovators migrate to Ubuntu Pro? {{ utm_campaign }}</h1>
+
+        <p class="u-no-padding--top p-heading--4">Get our e-book to find out</p>
+
+        <p>
+          <a href="#ubuntu-pro-ebook" class="p-button--positive">Download</a>
```

Comment:
We should add some context for a11y.
```suggestion
          <a href="#ubuntu-pro-ebook" class="p-button--positive">Complete the form to download<span class="u-off-screen"> the Why do Cloud innovators migrate to Pro? e-book</span></a>
```